### PR TITLE
Update python.props - releasing pyworker v2

### PR DESCRIPTION
**Pull request by vrdmr**
Release info at https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.2

### CODE REVIEW

File: build/python.props [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <Project>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="1.0.202005032" />
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="2.0.13908" />
```
Comment (fabiocav) [inline build/python.props:3]: Have we bumped the version on those to match the runtime major?

**Comment by ankitkumarr** [comment]
@vrdmr, you also need to update this -- github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/WebJobs.Script.Tests.csproj#L52

File: build/python.props (hunk 1 above)
Comment (fabiocav) [reply build/python.props:3]: Followed up offline. We can address later.

**Review by Hazhzeng — DISMISSED** [review]

**Comment by vrdmr** [comment]
Closing in favor of #6275

**Pull request closed without merging**